app/metrics.yaml

```diff
@@ -0,0 +1,18 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+toolbar.events:
+  url_committed:
+    type: event
+    description: >
+      Recorded when a URL is committed from the toolbar.
```

Comment:
This should be a very detailed description, listing off all possible values that this probe will send. This will be translated into documentation, so having the values sent will be useful a) for the data steward reviewing this to make sure no PII is being sent, and b) for the metrics team when they are looking at data to be able to quickly find what telemetry probes are related.

Could you add a little more detail here?

Reply:
@liuche In the latest commit, I added as much detail as I could about what is collected and when to the description field.

Reply:
@liuche sorry if I didn't mention before, this is ready for another look.  Thank you!

Reply:
@travis79 there's also a button next to each reviewer where you can "retrigger" a review - that way it'll show up in your reviewer's Review Requests dashboard on GH.

Reply:
Since I haven't been directly invited to this project, I don't have those buttons 😞.   Ping to @pocmo requesting an invite in case I need to do more work with the reference browser.